### DISCUSSION TITLE
Pobieranie oferty publicznej po ID

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#9556] [Suggestions - rejected]
Aktualnie metoda` /offers/listing` nie daje możliwości wyszukiwania oferty po ID.
Można to próbować obejść szukając po frazie i następnie sprawdzając id w wynikach, ale po ID byłoby idealnie.

**Comment by MaciejFrackowiak** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ aktualnie zasobu GET /offers/listing nie ma w publicznym API.
